### PR TITLE
Make term object

### PR DESCRIPTION
**Pull request by ninaprabhu**

### CODE REVIEW

File: src/main/java/com/google/sps/data/Term.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  int year;
+  String school;
+  ListIterator termIterator;
+  int numTerms;
+  int currTerm;
+  ArrayList<String> termList;
+
+  final List<String> QTR_SCHOOLS = Arrays.asList("caltech", "calpoly", "stanford", "berkeley");
+  final ArrayList<String> QTR_TERMS =
+      new ArrayList<String>(Arrays.asList("Winter", "Spring", "Fall"));
```
Comment (hailed748) [inline src/main/java/com/google/sps/data/Term.java:16]: what about summer?

File: src/main/java/com/google/sps/data/Term.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  final ArrayList<String> SEMESTER_TERMS = new ArrayList<String>(Arrays.asList("Spring", "Fall"));
+  // TODO: Potentially add in a summer term.
+
+  public Term(String termString, String termSchool) {
+    season = termString.split(" ")[0];
+    year = Integer.parseInt(termString.split(" ")[1]);
+    school = termSchool;
+    if (QTR_SCHOOLS.contains(school)) {
```
Comment (hailed748) [inline src/main/java/com/google/sps/data/Term.java:24]: QTR_SCHOOLS is a final, but what is new schools are added to the system?

Comment (ninaprabhu) [reply src/main/java/com/google/sps/data/Term.java:24]: This is just for testing - I'm going to find a list of all quarter system schools and put them there

File: src/main/java/com/google/sps/data/Term.java (hunk 1 above)
Comment (ninaprabhu) [reply src/main/java/com/google/sps/data/Term.java:16]: Wanted to discuss this with you actually - just sent a message in the chat about whether we should include summer or not

**Comment by Tetheta** [comment]
Can you ping me to review after you've resolved the merge conflicts?

**Comment by Tetheta** [comment]
Starting review

**Pull request merged**